src/app/shared/models/job.ts

```diff
@@ -0,0 +1,10 @@
+export interface Job {
+	id: string,
+	employerID: string,
+	title: string,
+	description: string,
+	requirements: string,
+	tags: string[],
+	period: string,
+	salary: string
```

Comment:
salary can be optional `salary?: string`

Reply:
fixed